examples/senseme_test.py

```diff
@@ -0,0 +1,630 @@
+from senseme import SenseMe
+from senseme import discover
```

Comment:
Import order should be standard lib, blank line, third party modules, blank line, then your own modules. Also, the senseme imports can be on the same line rather than two separate lines.